### PR TITLE
Fix RSC1: Rest constructor should accept a token.

### PR DESCRIPTION
**Pull request by tcard**

### CODE REVIEW

File: ablySpec/RestClient.swift [hunk 1]
```diff
@@ -44,7 +46,10 @@ class RestClient: QuickSpec {
                     expect(publishTask.error?.code).toEventually(equal(40005), timeout:testTimeout)
                 }
 
-                pending("should accept a token") {
+                it("should accept a token") {
+                    ARTClientOptions.setDefaultEnvironment("sandbox")
+                    defer { ARTClientOptions.setDefaultEnvironment("sandbox") }
```
Comment (ricardopereira) [inline ablySpec/RestClient.swift]: Assigning again with `sandbox`?

Comment (mattheworiordan) [inline ablySpec/RestClient.swift]: Yup, not sure I follow what this is doing

**Comment by mattheworiordan** [comment]
Other than one comment, LGTM

File: ablySpec/RestClient.swift (hunk 1 above)
Comment (tcard) [inline ablySpec/RestClient.swift]: Oh, silly me. Of course, this is meant to restore the default `nil` value. Fixed at b30271d.

**Comment by ricardopereira** [comment]
:+1:

**Pull request merged**